transforms/angle-brackets/test.js

```diff
@@ -1043,3 +1043,17 @@ test('wallstreet.hbs', () => {
       "
   `);
 });
+
+test('attr-space', () => {
+  let input = `
+    <MyComp::Test @color={{"custom-2"}} @visible={{Group.item.isNew}} />
+    <MyComp @value={{value}} @cont={{this}} @class={{model.some-stuff-here}} />
+  `;
+
+  expect(runTest('attr-space.hbs', input)).toMatchInlineSnapshot(`
+      "
+          <MyComp::Test @color={{\\"custom-2\\"}} @visible={{Group.item.isNew}} />
```

Comment:
Should this be?
```hbs
<MyComp::Test @color="custom-2" @visible={{Group.item.isNew}} />
```
or
```hbs
<MyComp::Test @color={{"custom-2"}} @visible={{Group.item.isNew}} />
```

Why are the slashes added?

Reply:
Either `@color="custom-2"` or `@color={{"custom-2"}}` are equivalent, although `@color={{"custom-2"}}` is plainly redundant.

As for the slashes...thats because the output from the `transform` function escapes ("), so it's necessary for string matching but is not ultimately seen by the end user

I can add both versions of your test to show they will transform to "_valid and equal_" but may not look the same.

Reply:
@Alonski I have updated the test to show you that the source pass through largely (if at all) unmodified.